tasks/configure-gpg.yml

```diff
@@ -0,0 +1,156 @@
+---
+
+# install jq
+
+- name: "Install jq package"
+  package: 
+    name: "jq"
+  become: yes
+
+# check when GPG Space already exists
+
+- name: "Count GPG Spaces"
+  shell: > 
+    curl -s -X GET 
```

Comment:
I would prefer to use always the url ansible module, and set the fact in a different task, using jq or jinja2 filters.